httpsender/AddHeaderXForwardedForRandomIP.js

```diff
@@ -1,4 +1,4 @@
-// @author Ruffenach Timothée
+// The original script comes from the Fuzzer HTTP Processor section under the name random_x_forwarded_for_ip.js
```

Comment:
I was not saying to remove the name, just mention the other script. This is for maintenance purposes, if changes are done in one script they most likely need to be done in bot.